facts/1.7/centos-6-x86_64.facts

```diff
@@ -381,5 +381,6 @@
   "swapfree_mb": "1535.85",
   "uid_min": "500",
   "package_provider": "yum",
-  "service_provider": "redhat"
+  "service_provider": "redhat",
+  "augeasversion": "1.4.0"
```

Comment:
On a el6 ws in my dev environment, the augeasversion fact is 1.10.1 and there is no service_provider fact

Reply:
@lnemsick-simp `1.4.0` is relatively arbitrary but old enough for the oldest supported version of our stuff. The issue was that it was missing completely and that was causing problems. Only one module actually cares about the version and it sets it explicitly.

`service_provider` was already there from the olden days.